Follow one sentence per line (#292)
grammyjs/website: site/docs/plugins/ratelimiter.md

# Rate Limit Users (`ratelimiter`)

rateLimiter is a rate-limiting middleware for Telegram bots made with grammY or [Telegraf](https://github.com/telegraf/telegraf) bot frameworks.
In simple terms, it is a plugin that helps you deflect heavy spamming in your bots.
To understand rateLimiter better, you can take a look at the following illustration:

![rateLimiter's role in deflecting spam](/rateLimiter-role.png)

## How Does It Work Exactly?

Under normal circumstances every request will be processed and answered by your bot which means spamming it will not be that difficult.

> Each user might send multiple requests per second and your script has to process each request, but how can you stop it? with rateLimiter!

Reviewer: ```suggestion
Each user might send multiple requests per second and your script has to process each request, but how can you stop it?
With rateLimiter!
```

### About `storageClient`

The `MEMORY_STORE` or the in-memory tracking is suitable for most bots, however if you implement clustering for your bot you will not be able to use the in-memory storage effectively.
That's why the Redis option is provided as well.

> You can pass a Redis client from [ioredis](https://github.com/luin/ioredis) or [redis](https://deno.land/x/redis) in case you use deno.

Reviewer: ```suggestion
You can pass a Redis client from [ioredis](https://github.com/luin/ioredis) or [redis](https://deno.land/x/redis) in case you use Deno.
```

> In reality any Redis driver that implements the `incr` and `pexpire` methods should work just fine.

Reviewer: ```suggestion
In reality, any Redis driver that implements the `incr` and `pexpire` methods should work just fine.
```

### Default Configuration

> The following example uses [express](https://github.com/expressjs/express) as the webserver and [webhooks](https://grammy.dev/guide/deployment-types.html) for our small bot.

Reviewer: ```suggestion
The following example uses [express](https://github.com/expressjs/express) as the webserver and [webhooks](/guide/deployment-types.md) for our small bot.
```

Author: I don’t think we need the ".md".
